src/app/pagamentoplanos/service/script.store.ts

```diff
@@ -0,0 +1,8 @@
+interface Scripts {
+    name: string;
+    src: string;
+}  
+export const ScriptStore: Scripts[] = [
+    {name: 'MercadoPago', src: 'https://sdk.mercadopago.com/js/v2'},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8398929)
```suggestion
    {name: "MercadoPago", src: 'https://sdk.mercadopago.com/js/v2'},
```